core/config/config.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package config;
+
+import "chain/protocol/bc/hash.proto";
+
+message Config {	
+	string id = 1;
+	bool is_signer = 2;
+	bool is_generator = 3;
+	bc.ProtoHash blockchain_id = 4;
+	string generator_url = 5;
+	string generator_access_token = 6;
+	string block_hsm_url = 7;
+	string block_hsm_access_token = 8;
+	uint64 configured_at = 9;
+	string block_pub = 10;
+	repeated BlockSigner signers = 11;
+	uint32 quorum = 12;
+	uint64 max_issuance_window = 13;
```

Comment:
This reminds me, it would be good to double check that the generated type is actually compatible. Protoc generates json tags, but I am not 100% sure that all the conventions match (e.g. decoding hex vs base64). The BlockSigner pubkey field in particular might be problematic.